engine-src/Game/LambdaHack/Client/UI/HandleHumanGlobalM.hs

```diff
@@ -1058,33 +1018,97 @@ verifyEscape = do
     then failWith "here's your chance!"
     else return $ Right ()
 
--- | Guess and report why the bump command failed.
-guessAlter :: COps -> [TriggerTile] -> ContentId TileKind -> Text
-guessAlter COps{cotile} (TriggerTile{ttfeature=TK.OpenTo _} : _) t
-  | Tile.isClosable cotile t = "already open"
-guessAlter _ (TriggerTile{ttfeature=TK.OpenTo _} : _) _ = "cannot be opened"
-guessAlter COps{cotile} (TriggerTile{ttfeature=TK.CloseTo _} : _) t
-  | Tile.isOpenable cotile t = "already closed"
-guessAlter _ (TriggerTile{ttfeature=TK.CloseTo _} : _) _ = "cannot be closed"
-guessAlter _ _ _ = "never mind"
-
 -- * AlterWithPointer
 
 -- | Try to alter a tile using a feature under the pointer.
 alterWithPointerHuman :: MonadClientUI m
-                      => [TriggerTile] -> m (FailOrCmd RequestTimed)
-alterWithPointerHuman ts = do
-  COps{corule=RuleContent{rXmax, rYmax}, cotile} <- getsState scops
-  lidV <- viewedLevelUI
+                      => m (FailOrCmd RequestTimed)
+alterWithPointerHuman = do
+  COps{corule=RuleContent{rXmax, rYmax}} <- getsState scops
   -- Not @ScreenContent@, because not drawing here.
```

Comment:
Actually, this comment belongs a line above (using RuleContent instead of ScreenContent, because it's checking for logical move conditions, not checking where on the UI screen, to paint things).